scipy/optimize/tests/test_linprog.py

```diff
@@ -1659,6 +1659,24 @@ def test_bug_8664(self):
                           method=self.method, options={"presolve": False})
         assert_(not res.success, "Incorrectly reported success")
 
+    def test_bug_11617(self):
```

Comment:
I'd suggest parameterizing the test rather than including the `for` loop: https://docs.pytest.org/en/latest/parametrize.html